manifests/j/jely2002/youtube-dl-gui/2.3.1/jely2002.youtube-dl-gui.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: jely2002.youtube-dl-gui
+PackageVersion: 2.3.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
```

Comment:
`InstallerType: nullsoft` This is listed twice

Reply:
This should not affect the functionality. Will be fixed in one of the next updates eventually.